internal/pkg/project/manifest/manifest_test.go

```diff
@@ -161,92 +152,12 @@ func TestManifestValidateNestedField(t *testing.T) {
 			),
 		},
 	})
-	err := m.validate()
+	err := content.validate()
 	assert.Error(t, err)
 	expected := "manifest is not valid:\n  - key=\"branches[0].id\", value=\"0\", failed \"required\" validation"
 	assert.Equal(t, expected, err.Error())
 }
 
-func TestIsObjectIgnored(t *testing.T) {
```

Comment:
Toto je presunute v dalsom commite do `model.Filter` + tests.
Nejak to tu bolo omylom.